src/js/components/locations-configuration/LocationDetails.jsx

```diff
@@ -279,117 +290,141 @@ class LocationDetails extends Component {
+                      active: values.active,
+                      debouncedLocationGroupsFetch: this.debouncedLocationGroupsFetch,
+                      debouncedOrganizationsFetch: this.debouncedOrganizationsFetch,
+                      debouncedUsersFetch: this.debouncedUsersFetch,
+                      openNewOrganizationModal: this.openNewOrganizationModal,
+                      openNewLocationGroupModal: this.openNewLocationGroupModal,
                     }),
-                  }}
-                  locationTypes={this.state.locationTypes}
-                />
-                <div className="d-flex w-100 ml-1 pt-2 justify-content-between align-items-center">
-                  <Checkbox
-                    id="useDefaultActivities"
-                    value={this.state.useDefaultActivities}
-                    onChange={val => this.setState({ useDefaultActivities: val })}
-                    withLabel
-                    label={this.props.translate('react.locationsConfiguration.useDefaultActivities.label', 'Use default settings for Supported Activities')}
-                  />
-                  <button
-                    type="button"
-                    className="btn btn-primary btn-xs"
-                    onClick={() => {
-                      this.setState({ useDefaultActivities: true });
-                      resetSupportedActivities(values.locationType);
-                    }}
-                  >
+                  )}
+
+                  <div className="form-title"><Translate id="react.locationsConfiguration.typeAndActivities.label" defaultMessage="Location Type and Supported Activities" /></div>
+                  <div className="form-subtitle">
                     <span>
-                      <i className="fa fa-refresh pr-2" />
-                      <Translate id="react.locationsConfiguration.resetToDefault.label" defaultMessage="Reset to default settings" />
+                      <Translate id="react.locationsConfiguration.typeAndActivitiesDescription.label" />&nbsp;
+                      <a target="_blank" rel="noopener noreferrer" href="https://openboxes.atlassian.net/wiki/spaces/OBW/pages/1744633857/Location+Types+and+Supported+Activities">
+                        <Translate id="react.locationsConfiguration.here.label" defaultMessage="here" />
+                      </a>.
                     </span>
-                  </button>
-                </div>
-                <div className="location-supported-activities">
+                  </div>
+
                   <SelectField
-                    fieldName="supportedActivities"
+                    fieldName="locationType"
                     fieldConfig={{
+                      label: 'react.locationsConfiguration.locationType.label',
+                      defaultMessage: 'Location Type',
                       attributes: {
                         multi: true,
```

Comment:
why you added the multi here?

Reply:
this was probably a mistake during conflict resolution